src/faas/v0/context.ts

```diff
@@ -176,9 +185,10 @@ export class HttpContext extends TriggerContext<HttpRequest, HttpResponse> {
       status: 200,
       headers: {},
       body: '',
+      json: jsonResponse(ctx),
```

Comment:
Would be good to do a call on this if you've got some time for a code walkthrough.